samples/smart-home/client/src/app/features/chat/chat-panel.component.ts

```diff
@@ -201,26 +203,26 @@ export class ChatPanelComponent {
           lightId: s.string('The id of the light'),
         },
       }),
-      exposeComponent(CardComponent, {
-        name: 'card',
-        description: 'Show a card to the user',
-        children: 'any',
-        props: {
-          title: s.streaming.string('The title of the card'),
-        },
-      }),
+      // exposeComponent(CardComponent, {
```

Comment:
just confirming we want to keep this commented out

Reply:
Nah, we don't. I was trying to find the specific failure point in the Palmyra models generating against the UI schema. I was hoping the `CardComponent` was the point of failure since it is the only component in this schema that accepts any component as a child.